README.rst

```diff
@@ -73,6 +73,8 @@ What's New?
 in development
 ^^^^^^^^^^^^^^
 
+* When option ``-W/--warnings-as-errors`` is used, all warnings issued with the warnings module now makes pydoctor exit with code 3.
```

Comment:
It's actually only warnings that pydoctor issues, not "all warnings issued with the warnings module". Perhaps:

```suggestion
* When option ``-W/--warnings-as-errors`` is used, pydoctor exits with code 3 when it issues any warning.
```